app/models/step.rb

```diff
@@ -10,20 +12,17 @@ class Step < ApplicationRecord
   # FIXME: fill in position from id!
 
   default_scope { order(position: :asc) }
+  scope :published, -> do
```

Comment:
Toto nie je nejako extra dlhe v klude to daj do `-> { }` notacie

Reply:
zmenene, tento scope uz nepotrebujeme